### DISCUSSION TITLE
tsconfig.json

### DISCUSSION THREAD

**Discussion by justindasilva** [nuxt/nuxt#18303] [Questions] [answered]
I have a `tsconfig.json` file in my root directory that looks like this:

```
{
  "extends": "./.nuxt/tsconfig.json",
  "compilerOptions": {
    "esModuleInterop": false,
    "allowSyntheticDefaultImports": true,
    "skipLibCheck": true,
    "strict": false,
    "noUnusedLocals": true,
    "types": [
      "node",
    ]
  }
}
```

VSCode is showing me an error that it cannot read `./nuxt/tsconfig.json`

The file does exist within the `.nuxt` folder.

So not sure what's going on.

**Comment by sduduzog** [comment, marked as answer]
It's auto generated when you call `yarn dev`/`npm run dev` given your script is setup to run `nuxi dev`. See https://v3.nuxtjs.org/getting-started/commands

**Reply by justindasilva** [reply]
So funny enough, my issue was fixed when I restarted VSCode :)
